storage/remote/otlptranslator/prometheusremotewrite/number_data_points.go

```diff
@@ -114,15 +114,10 @@ func (c *PrometheusConverter) addSumNumberDataPoints(ctx context.Context, dataPo
 				return nil
 			}
 
-			createdLabels := make([]prompb.Label, len(lbls))
-			copy(createdLabels, lbls)
-			for i, l := range createdLabels {
-				if l.Name == model.MetricNameLabel {
-					createdLabels[i].Value = name + createdSuffix
-					break
-				}
-			}
-			c.addTimeSeriesIfNeeded(createdLabels, startTimestamp, pt.Timestamp())
+			c.builder.Reset(lbls)
+			// add created suffix to the metric name
```

Comment:
```suggestion
			// Add created suffix to the metric name for CT series.
```

BTW I wished this was not added - those series are not very useful, only increasing cardinality for little reason, is it optional or by default?

Reply:
It is optional.  Also seems particularly pointless now that we have CT in PRW 2.0 lol.